src/AspNet.Security.OAuth.Introspection/Events/AuthenticationFailedContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
```

Comment:
Much of that is done automatically by Resharper or VS as needed - Now you're just being picky :stuck_out_tongue_winking_eye: